### ISSUE TITLE
Can not start client in dev

### ISSUE THREAD

**Issue by da1z** [resend/react-email#947]
### Describe the Bug

Cloned repo

yarn install
yarn dev

cd client

yarn dev

going to localhost:3000
getting error 

```
Failed to compile
../packages/render/dist/index.mjs
Error: 
  x NEXT_RSC_ERR_SERVER_IMPORT: react-dom/server
    ,-[/Users/kirylchetyrbak/Projects/react-email/packages/render/dist/index.mjs:26:1]
 26 | var __forAwait = (obj, it, method) => (it = obj[__knownSymbol("asyncIterator")]) ? it.call(obj) : (obj = obj[__knownSymbol("iterator")](), it = {}, method = (key, fn) => (fn = obj[key]) && (it[key] = (arg) => new Promise((yes, no, done) => (arg = fn.call(obj, arg), done = arg.done, Promise.resolve(arg.value).then((value) => yes({ value, done }), no)))), method("next"), method("return"), it);
 27 | 
 28 | // src/render.ts
 29 | import * as ReactDomServer from "react-dom/server";
    : ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 30 | import { convert } from "html-to-text";
 31 | import pretty from "pretty";
 32 | var render = (component, options) => {
    `----

  x NEXT_RSC_ERR_SERVER_IMPORT: react-dom/server
    ,-[/Users/kirylchetyrbak/Projects/react-email/packages/render/dist/index.mjs:53:1]
 53 | // src/renderAsync.ts
 54 | import { convert as convert2 } from "html-to-text";
 55 | import pretty2 from "pretty";
 56 | import { renderToReadableStream, renderToStaticMarkup as renderToStaticMarkup2 } from "react-dom/server";
```

### Which package is affected (leave empty if unsure)

client

### Link to the code that reproduces this issue

this repo in branch main

### To Reproduce

Cloned repo

yarn install
yarn dev

cd client

yarn dev

going to localhost:3000

### Expected Behavior

no errors

**Comment by kicksent** [comment]
Hey da1z, I also had a poor experience with using yarn. However, after about 30 minutes the dev server did start. I am using an M2 Macbook Pro.

**Comment by antoinersw** [comment]
Hello I have the same problem using yarn. The weird this is i can browse to the homepage but when i click on an email it crashes as such.I m using tailwind too. It was working fine last week

**Comment by qiuxan** [comment]
I have the same issue here, the same as described by @antoinersw

**Comment by gregavola** [comment]
The problem is that now is not compatible with Next 14.

@zenorocha any ideas here?

**Comment by mkreuzmayr** [comment]
Also experiencing the same issues when using Next.js 14 in the parent project.

**Comment by gabrielmfern** [comment]
This should now be fixed with `react-email@2.0.0`.
